cmd/lifecycle/analyzer.go

```diff
@@ -65,6 +65,7 @@ func (a *analyzeCmd) Args(nargs int, args []string) error {
 }
 
 func (a *analyzeCmd) Privileges() error {
+	a.setEnvRegistryAuth()
```

Comment:
We considered exporting this function and making it part of the interface, but not every phase needs it, so we did it this way.

Reply:
I like it as is. `Privileges` still feels like the right high-level abstraction to me.